README.md

```diff
@@ -175,6 +175,16 @@ $ nba-go player Curry -p
 ```
 ![player -p gif](https://user-images.githubusercontent.com/12113222/32500032-234e8fba-c40f-11e7-87c0-6e42a66a52dc.gif)
 
+##### `-c` or `--compare`  
+Get and compare the stats from multiple players. The better stat will be highlighted in green to make comparing easier.
+When listing the multiple names they must be in quotes and seperated by commas. Can be combined with the -i, -r, and -p flags.
+
+```
+$ nba-go player "Lebron James, Stephen Curry, James Harden" -c -i -r -p
+```
+![player -c gif](https://user-images.githubusercontent.com/36014115/37246796-0d4a2d26-2465-11e8-89d5-4f7f33b4ef59.gif)
```

Comment:
I think this gif file could be done by myself, because  it would be better if all gif files are recorded by using same terminal which is [hyper](https://github.com/zeit/hyper).